docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[releasenotes]]
+== Release Notes
+
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-3-0,Logstash 5.3.0>>
+
+[[logstash-5-3-0]]
+=== Logstash 5.3.0 Release Notes
+
+
+* Persistent queues:
+    ** Changed the default queue location on disk to include the pipeline's ID in the path hierarchy.
+       By default, the queue is now created under <path.data>/queue/main. This breaking change was made to
+       accommodate an upcoming feature where multiple, isolated pipelines could be run on the same Logstash
+       instance.
+    ** Added a _recovery_ process that runs when Logstash starts up to recover data written to the persistent
+       queue, but not yet checkpoint'd. This is useful in a situation when the input had written data to the queue,
+       but Logstash crashed before writing to the checkpoint file.
+    ** Added exclusive access to the persistent queue on disk, as defined by the path.queue setting. Using a file
+       lock we ensure that only a single Logstash instance has access to write to the queue on the same path, to
+       guard against corruption ({lsissue}6604[Issue 6604]).
+    ** You can now safely reload the pipeline config when using persistent queues. Previously, the reloading of
+       config could result in data corruption.  In 5.3, the reload sequence has been changed to reliably shutdown the
```

Comment:
Change to: "...has been changed to reliably shut down the..."
shut down is the verb form
shutdown is the noun form